src/v/kafka/server/protocol.cc

```diff
@@ -100,6 +100,13 @@ ss::future<> protocol::apply(rpc::server::resources rs) {
     return ss::do_until(
              [ctx] { return ctx->is_finished_parsing(); },
              [ctx] { return ctx->process_one_request(); })
+      .handle_exception([ctx](std::exception_ptr propogate) {
+          vlog(
+            klog.info,
+            "Auth failed on ip: {}",
```

Comment:
what kinda auth? can you be more specific here.

Reply:
What else can you print about the context that would be helpful to a sysadmin looking at the log.

Reply:
Currently, looking into other information that I can capture from the exception and client_host. I'll push changes with all that I can find.